backend/src/model/dto.rs

```diff
@@ -53,11 +53,17 @@ pub struct NewSeedDto {
 
 #[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.
```

Comment:
Please remove lines `#[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.` completely for all newly created enums/structs.
Instead you should document the structs and fields with code comments e.g. 
```rust
/// This represents ... and is used for ...
#[typeshare]
#[derive(Debug, Serialize, PartialEq, Eq, Deserialize, ToSchema)]
pub struct PlantsSummaryDto {
    /// The id of the entity
    pub id: i32,
   /// <Explaination of the field>
    pub binomial_name: String,
   /// <Explaination of the field>
    pub common_name: Option<Vec<Option<String>>>,
}
```

Please write comments that can be used to fully understand what the structs/fields do and what might be special about them. Those comments will later on be part of the documentation (generated from code).

You can look at the [doc of Vec](https://doc.rust-lang.org/std/vec/struct.Vec.html) and the [corresponding code](https://doc.rust-lang.org/src/alloc/vec/mod.rs.html#151) as and example. The doc is generated from code.